src/components/sectionList/filters/filterSelectors/PublicAccessFilter.tsx

```diff
@@ -0,0 +1,19 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import { ConstantSelectionFilter } from './ConstantSelectionFilter'
+
+const constants = {
+    'rw------': 'Public can edit',
+    'r-------': 'Public can view',
+    '--------': 'Public cannot access',
+}
```

Comment:
Note that this currently worldn't work for objects that also have "data sharing" (like category options). This was just a simple implementation for now.